var/spack/repos/builtin/packages/r-combinat/package.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RCombinat(RPackage):
+    """routines for combinatorics"""
+
+    homepage = "https://cran.r-project.org/package=combinat"
+    url      = "https://cran.r-project.org/src/contrib/combinat_0.0-8.tar.gz"
+
```

Comment:
please add list_url

Reply:
why is list_url needed here?

Reply:
when new versions are released on cran, old ones are moved to Archive, and without the list_url, spack will fail to retrieve this version when that happens.